index.html

```diff
@@ -8,8 +8,39 @@
 <div class="block">
   <h1><span class="header-inside">Projects</span></h1>
   <div id="projects-panel">
+    <table class="projects">
+      <noscript>
+        {% for p in site.data.projects %}
+        {% assign project = p[1] %}
```

Comment:
Just to check, do you need that array index? IIRC, `site.data.projects` is already an array of projects, and your forloop should be going through each project iteratively - what you've got looks like it's an array of arrays.